app/controllers/tweets_controller.rb

```diff
@@ -0,0 +1,74 @@
+class TweetsController < ApplicationController
+
+	get '/tweets' do
+		if logged_in?
+			@tweets = Tweet.all
+			@user = User.find(session[:user_id])
+			erb :'/tweets/index'
+		else
+			redirect "/login"
+		end
+	end
+
+	post '/tweets' do
+    if params[:content] != ""
+      @tweet = Tweet.new(content: params[:content])
```

Comment:
The best way to do this is to make use of the `has_many` relationship on User:
```
@tweet = current_user.tweets.create(content: params[:content])
```
Both approaches are good, but in some cases you may find this option useful.